pipeline/customer/customer.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import argparse
+
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions, SetupOptions
+
+from pipeline.common import common
+from pipeline.customer import detail
+
+
+def run(argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--detail_input',
+        type=str,
+        help='Input file pattern for the customer details',
+        required=True)
+
+    known_args, pipeline_args = parser.parse_known_args(argv)
```

Comment:
BTW: This way of doing it should even be compatible with Flink. Why did we do it in our code with the `@classmethod` instead?

Reply:
Interesting. Did not know that. Just copied it from the examples.